<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryPolicyDialog</class>
 <widget class="QDialog" name="MemoryPolicyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>内存使用策略</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="4">
    <widget class="QSpinBox" name="sb_self_limit">
     <property name="suffix">
      <string> Gib时自动缓存</string>
     </property>
     <property name="prefix">
      <string>当本进程占用&gt; </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16384</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QDoubleSpinBox" name="dsb_system_limit">
     <property name="prefix">
      <string>当系统剩余内存&lt; </string>
     </property>
     <property name="suffix">
      <string> %时自动缓存</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>5.000000000000000</double>
     </property>
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>提示：以上只是缓存数据的目标，不能保证内存占用一定小于设置的上限。缓存数据的实质是将构建的三维结构暂存硬盘（并释放相应内存），需要时再读取，因此必然使导出地图画变慢。</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QCheckBox" name="cb_auto_cache">
     <property name="text">
      <string>自动缓存数据以节约内存</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pb_ok">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pb_cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pb_reset">
     <property name="text">
      <string>恢复默认</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
